### PR TITLE
Added API Endpoints

### PR DESCRIPTION
**Pull request by KetanBansal**

### CODE REVIEW

**Comment by mohan7427** [comment]
Review done

File: API_Endpoints/crud.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from time import sleep
+from fastapi import FastAPI, HTTPException, Body, responses, status
+from statistics import mode
+from sqlalchemy.orm import Session
+import models
+
+def get_clusters(session: Session, skip: int = 0, limit: int = 100):
```
Comment (mohan7427) [inline API_Endpoints/crud.py]: function name should be relevant to the project

File: API_Endpoints/crud.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    return session.query(models.Boost).offset(skip).limit(limit).all()
+
+
+def get_cluster_by_id(session: Session, job_id: int):
```
Comment (mohan7427) [inline API_Endpoints/crud.py]: same as above

File: API_Endpoints/models.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import Column, Integer, String
+from database import Base
+
+class Boost(Base):
+    __tablename__ = 'Storage'
+    JobId = Column(Integer, primary_key=True, index=True)
+    BlockSize = Column(String(256))
```
Comment (mohan7427) [inline API_Endpoints/models.py:7]: Shouldn't we also require the below entries in the table?

```
name=<run1>
ioengine=libaio,rbd,rados
Diskname=/dev/sdb:/dev/sdc
bs=1k:4m,all
iodepth=32
numjobs=4
rw=read,write,randread,randwrite,mix,rwmixread-50,rwmixread-70
runtime
```

**Review by ishita2301 — APPROVED** [review]

**Review by mohan7427 — APPROVED** [review]

**Pull request merged**